app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -280,21 +280,22 @@ def update_form_with_classification_code(classification_code)
   end
 
   def log_max_cfi_metrics_on_submit
-    user = User.find(user_uuid)
-    max_cfi_enabled = Flipper.enabled?(:disability_526_maximum_rating, user) ? 'on' : 'off'
-    ClaimFastTracking::DiagnosticCodesForMetrics::DC.each do |diagnostic_code|
-      next unless max_rated_diagnostic_codes_from_ipf.include?(diagnostic_code)
-
+    any_claimed = false
+    max_rated_diagnostic_codes_from_ipf.each do |diagnostic_code|
       disability_claimed = diagnostic_codes.include?(diagnostic_code)
-
-      if disability_claimed
-        StatsD.increment("#{MAX_CFI_STATSD_KEY_PREFIX}.#{max_cfi_enabled}.submit.#{diagnostic_code}")
-      end
-      Rails.logger.info('Max CFI form526 submission',
-                        id:, max_cfi_enabled:, disability_claimed:, diagnostic_code:,
-                        cfi_checkbox_was_selected: cfi_checkbox_was_selected?,
-                        total_increase_conditions: increase_disabilities.count)
+      any_claimed ||= disability_claimed
```

Comment:
I wonder if it'd be more readable to separate it out into something like

```ruby
any_claimed = (max_rated_diagnostic_codes_from_ipf & diagnostic_codes).any?
```

Or even combine it with the `num_max_rated_cfi` value below.